test/purchases_flutter_test.dart

```diff
@@ -76,17 +83,26 @@ void main() {
     );
   });
 
-  test('addCustomerInfoUpdateListener does not call listener if it does not have one', () async {
+  test(
+      'addCustomerInfoUpdateListener does not call listener if it does not have one',
```

Comment:
Whoops, I'm guessing we're going over the line limit. Should this be caught by the linter?

Reply:
this is interesting, idk why but the linter didn't catch this. When I ran `pub publish --dry-run` it formatted these and also gave me some errors from `dart analyze`. I ran `dart analyze` though and got so many other issues that I think `pub publish` is using both a dart analyze and linter custom configuration